src/wallet/wallet.h

```diff
@@ -607,16 +607,21 @@ struct CoinSelectionParams
     size_t change_output_size = 0;
     size_t change_spend_size = 0;
     CFeeRate effective_fee = CFeeRate(0);
+    CFeeRate m_long_term_feerate = CFeeRate(0);
+    CFeeRate m_discard_feerate = CFeeRate(0);
```

Comment:
`CFeeRate` default constructor gives it a 0, so this is not needed in the declaration
```suggestion
    CFeeRate m_long_term_feerate;
    CFeeRate m_discard_feerate;
```

Reply:
done